File: backend/BusinessService/Common/CmGlobal.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: 
// 
// 作成日　　: 2022.12.12
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.Service
{
    public class CmGlobal
    {
        private static EnumRuntimeMode _RuntimeMode = EnumRuntimeMode.NORMAL;

        public static EnumRuntimeMode RuntimeMode
        {
            get
            {
                return _RuntimeMode;
            }
            set
            {
                _RuntimeMode = value;
                // DaGlobal.WriteExcelLog = Not (_RuntimeMode = SvEnumRuntimeMode.NORMAL)
            }
        }
    }
}

File: backend/JbdGjsService/Common/CmGlobal.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: 
' 
' 作成日　　: 2024.07.12
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Service
    Public Class CmGlobal
        Private Shared _RuntimeMode As EnumRuntimeMode = EnumRuntimeMode.NORMAL

        Public Shared Property RuntimeMode As EnumRuntimeMode
            Get
                Return _RuntimeMode
            End Get
            Set(value As EnumRuntimeMode)
                _RuntimeMode = value
            End Set
        End Property
    End Class
End Namespace
